script/SetCancellers.s.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+import {OptimismGovernor} from "../src/OptimismGovernor.sol";
+import {TimelockControllerUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/TimelockControllerUpgradeable.sol";
+import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import {AccessControlEnumerableUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/access/AccessControlEnumerableUpgradeable.sol";
+
+/**
+ * @title SetCancellers
+ * @notice Script to add three Optimism Foundation L2 Safes to the canceller role
+ * for the timelock, allowing them to cancel queued transactions if a malicious
+ * or erroneous proposal is discovered. Proposers will retain their canceller role.
+ * @dev Run this script with:
+ *      forge script script/SetCancellers.s.sol:SetCancellers --rpc-url $RPC_URL --broadcast --verify
+ */
+contract SetCancellers is Script {
+    address constant GOVERNOR_ADDRESS = 0x0000000000000000000000000000000000000000; // Replace with actual Optimism Governor address
+    address constant L2_SAFE_1 = address(0); // Replace with first L2 Safe address
+    address constant L2_SAFE_2 = address(0); // Replace with second L2 Safe address
+    address constant L2_SAFE_3 = address(0); // Replace with third L2 Safe address
+
+    // Constants
+    bytes32 constant PROPOSER_ROLE = keccak256("PROPOSER_ROLE");
+    bytes32 constant CANCELLER_ROLE = keccak256("CANCELLER_ROLE");
+    bytes32 constant TIMELOCK_ADMIN_ROLE = keccak256("TIMELOCK_ADMIN_ROLE");
+
+    function run() external {
+        // Boot deployer private key
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
Similar to the bash script we should be using the keystore to manage private keys.
```suggestion
```